spec/requests/custom_button_sets_spec.rb

```diff
@@ -0,0 +1,172 @@
+RSpec.describe 'CustomButtonSets API' do
+  let(:cb_set) { FactoryGirl.create(:custom_button_set, :name => 'custom_button_set') }
+  let(:cb_set2) { FactoryGirl.create(:custom_button_set, :name => 'custom_button_set2') }
```

Comment:
I don't see this name being referenced anywhere, so can be omitted

Reply:
I'm using `cb_set2` in the bulk delete spec... so keeping it :)